docs/docs/naming.md

```diff
@@ -26,11 +26,36 @@ $ dig search.marathon.mesos
 search.marathon.mesos.		60	IN	A	10.9.87.94
 ```
  
+## Container IP A Records
+
+The A records above point to the host IP of a given service. If container IPs are supported by Mesos and the executor of a test, Mesos-DNS will create another A record for each of the A records above by prefixing it with `_container` and returning the container IP:
```

Comment:
Thanks for pointing this out. I am not really sure how to formulate this. We use `_underscore` to avoid name clashes. But is there a danger that any client will refuse such a name? And if yes, what would be your suggestion? Something like `container.search.marathon.mesos`?

We can add a note like this:
```
*Note*: According to the original [RFC1035](https://www.ietf.org/rfc/rfc1035.txt) underscores are not allowed in A records. [RFC2181](https://www.ietf.org/rfc/rfc2181.txt) section 1 clarifies that underscores (and in fact any binary string) must be allowed by DNS servers. We use the underscore here to avoid naming clashes. The `_container` names are not meant to be hostnames (which forbit underscores as well).
```

But this feels like hand waving instead of a rigorous technical answer.